src/runtime/Uno.AppLoader-MonoMac/monostub.m

```diff
@@ -1,4 +1,6 @@
-//clang -m64 monostub.m -o monostub -framework AppKit -mmacosx-version-min=10.6
+// We include a precompiled monostub to make compilation of solution possible on non-mac platforms,
+// so if this file is modified, it needs to be manually recompiled:
+//     clang -m64 monostub.m -o monostub -framework AppKit -mmacosx-version-min=10.6
```

Comment:
Maybe this should be added as `src/runtime/Uno.AppLoader-MonoMac/README.md` instead, to increase visibility? Just a thought...

Reply:
I can add a `README.md` file too. Howver, it's not given that someone trying to change `monostub.m` will read the README, so keeping the comment to avoid any surprises.